pkgs/by-name/ai/airwin2rack/package.nix

```
@@ -0,0 +1,215 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+  makeDesktopItem,
+  copyDesktopItems,
+  srcOnly,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  xorg,
+  freetype,
+  libGLU,
+  libjack2,
+  juce,
+  webkitgtk_4_0,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libxkbcommon,
+  libdatrie,
+  libepoxy,
+  libsoup_2_4,
+  lerc,
+  sqlite,
+  glib,
+  gtk3-x11,
+  curl,
+  enableVCVRack ? false,
+}:
+let
+  clapJuceExtensions = fetchFromGitHub {
+    owner = "free-audio";
+    repo = "clap-juce-extensions";
+    rev = "4f33b4930b6af806018c009f0f24b3a50808af99";
+    hash = "sha256-M+T7ll3Ap6VIP5ub+kfEKwT2RW2IxxY4wUPRQKFIotk=";
+    fetchSubmodules = true;
+  };
+
+  vcvRackSdk = if enableVCVRack then (srcOnly pkgs.vcv-rack) else null;
+  pname = "airwin2rack";
+  version = "2.13.0";
```

There's no version 2.13.0 upstream, as far as I can tell.

If this is meant to be some development version past 2.12.0, it ought to be
```suggestion
  version = "2.12.0-unstable-2024-12-22";
```

I would recommend using a stable version though, as `fetchgit` has inherent limitations when it comes to fetching by commit hash (as opposed to tag or other ref)

---

The project file specified that it is indeed 2.13

---

https://github.com/baconpaul/airwin2rack/blob/main/plugin.json#L4

The version of the plugin is 2.13.0, and the specified github release is "DAW Plugin"

---

Oh, OK, that's awkward.